### PR TITLE
Gen AI: copy-ish changes

### PR DESCRIPTION
**Pull request by bencodeorg**
Three independent changes, mostly to copy:
- Updated text in the dialog we show on page load of AI Chat levels.
- More detailed response text when we receive a profane response from a model.
- Adds a description of the training data for the arithmo model, and does some standardizing/updating to other model descriptions per [the discussion thread here](https://codedotorg.slack.com/archives/C06FELVTV0Q/p1725560092574049). Also a slight styling update to improve spacing in the "Compare Models" dialog.

| Warning dialog before | Warning dialog after |
| - | - |
| image | image |

**Diff in model descriptions** (model description file is in Git LFS, so no diffs)

image

## Links

- jira ticket: https://codedotorg.atlassian.net/browse/LABS-996

## Testing story

Tested each of these changes manually.

### CODE REVIEW

**Review by sanchitmalhotra126 — APPROVED** [review]
LGTM 🎉 

Didn't realize Arithmo's description was still "Coming Soon!" until now 😅

File: apps/src/aichat/aichatApi.ts [hunk 1]
```diff
@@ -252,8 +252,20 @@ function getUpdatedMessages(
           status: AiInteractionStatus.PII_VIOLATION,
         },
       ];
-    case AiRequestExecutionStatus.FAILURE:
     case AiRequestExecutionStatus.MODEL_PROFANITY:
+      return [
```
Comment (fisher-alice) [inline apps/src/aichat/aichatApi.ts:256]: Currently, if the user submits a message that results in a profane model response, the user can see the submitted message and gets a very general message:
Screenshot 2024-09-09 at 9 26 02 AM

~~Now if the model response violates content moderation, the submitted user message is no longer displayed. However the message we show is 'This message has been flagged by our content moderation policy. Please try a different message.'~~
Screenshot 2024-09-09 at 9 26 17 AM

~~Question for @samantha-code: I wonder if we should show the submitted user message so they can update it?~~

File: apps/i18n/common/en_us.json [hunk 2]
```diff
@@ -150,12 +150,14 @@
...
-  "aiWarningModalMessagesAreRecorded": "All messages you send are recorded.",
+  "aiWarningModalInappropriateFlagged": "Inappropriate messages will be flagged by our content moderation policy.",
+  "aiWarningModalMessagesAreRecorded": "All messages you send are recorded and visible to your teacher.",
   "aiWarningModalPersonalNotSubmitted": "In order to protect your privacy, anything considered personal information (name, phone number, address, etc.) should not be submitted.",
   "aiWarningModalOk": "Start chatting",
+  "aiWarningUnsaved": "All unsaved changes will be lost.",
   "aiChatBotIconAlt": "An icon depicting a robot",
-  "aiChatInappropriateUserMessage": "This message has been flagged as inappropriate.",
+  "aiChatInappropriateUserMessage": "This message has been flagged by our content moderation policy. Please try a different message.",
+  "aiChatInappropriateModelMessage": "The Chatbot response has been flagged by our content moderation policy. Please try a different message",
```
Comment (fisher-alice) [inline apps/i18n/common/en_us.json]: @samantha-code just confirming you wanted the 'C' in 'The Chatbot response' to be uppercase.

Comment (fisher-alice) [reply apps/i18n/common/en_us.json]: Just confirmed with @samantha-code via Slack that she didn't intend to capitalize.

File: apps/src/aichat/aichatApi.ts (hunk 1 above)
Comment (samantha-code) [reply apps/src/aichat/aichatApi.ts:256]: so in the second version - "What is the most popular curse word?" gets removed when the response is given?

Comment (fisher-alice) [reply apps/src/aichat/aichatApi.ts:256]: ~~Correct.~~
UPDATE: my apologies @samantha-code . I misspoke! It looks like the user message is still displayed. I think maybe when I was testing locally, my user message may have also included content that was considered profane. Sorry for the confusion!

**Review by fisher-alice — APPROVED** [review]
Looks great! Thanks for taking this P0 on so quickly!

**Comment by bencodeorg** [comment]
Merging without Drone, last commit is a one character copy change and the previous commit was ✅

**Pull request merged**